release-notes/v1.3.0.md

```diff
@@ -18,3 +18,16 @@ The following table lists the known issues for HPE CSI Driver for Kubernetes v1.
+|CON-1260|csi.k8s|iSCSI CHAP username/password is not updated in HPENodeInfos custom resource|HPE CSI driver maintains iSCSI CHAP credentials within each HPENodeInfo custom resource instance. When driver is re-deployed with changes to iSCSI CHAP username or password, sometimes HPENodeInfo instances are not updated/re-created causing them to contain stale data. This causes failures when an application pod is scheduled onto the node with stale iSCSI CHAP username or password|Possible workarounds for this include:<br />1. Delete HPENodeInfos instances manually and re-deploy CSI driver to create them with updated values.<br />2. Edit HPENodeInfos to update iSCSI CHAP username/password manually with each instance. Password has to be base64 encoded value|
+|CON-1266|csi.k8s|Failure to edit PVC size and other volume properties at the same time|With the introduction of the HPE Volume Mutator from v1.3.0, users can now edit volume parameters through PVC annotations. But, when volume properties are edited along with size at the same time, both the CSI Resizer and the HPE Volume Mutator might attempt PVC updates(i.e status and conditions). This might cause errors with either one of them during update saying resource version has changed externally and update process might fail|Avoid editing volume properties through annotations along with size at the same time. Resize is a different operation which involves controller/node expansion triggered through CSI external resizer. It is recommended to perform resize (volume size edit) and editing the rest of the volume properties like limitIops, limitMbps, etc.. as seperate operations.
+
+The following table lists the known issues for HPE Primera/3PAR CSP for HPE CSI Driver release 1.3.0
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|| Replication | Pod mount during primary array down | If the primary array is down in a replicated array setup, and if the pod mount is attempted, it will fail.|It is recommended to edit `backend` in primary secret (using `kubectl apply -f <secret>`) to make it point to secondary array ip|
+|| Replication | Cannot add volume to RCG (Replication Copy Group) when it is in primary role on secondary|When the RCG is active on the secondary array, a new volume (PV) addition to it will not be allowed|None|
+|| Replication | Autocompute of CPG is disallowed for replicated volumes | When using replication it is a must to use `cpg` parameter in the storage class (SC)| None|
+|| Replication | Mandatory RCG policies required | If existing RCG group is used for `remoteCopyGroup` parameter in SC, please ensure auto_failover, path_management policies are set to avoid failures during writes and during switchover process.|None |
+|| Snapshot | Restore of snapshot created using `virtualCopyOf` parameter| Snapshot created using `virtualcopyOf` option cannot be restored using CSI driver. That is to be done using cli command `promotesv`|None|
```

Comment:
Snapshot created using `virtualcopyOf` option <= virtualCopyOf <= Capital C

Reply:
Agreed, will update it